glib/src/object.rs

```diff
@@ -1783,7 +1783,7 @@ pub trait ObjectExt: ObjectType {
     /// Limits the lifetime of `closure` to the lifetime of the object. When
     /// the object's reference count drops to zero, the closure will be
     /// invalidated. An invalidated closure will ignore any calls to
-    /// [`Closure::invoke`](crate::Closure::invoke).
+    /// [`invoke`](crate::closure::RustClosure::invoke).
```

Comment:
Normally you use only the method name if it's on the same type which is not the case here